src/Features/LanguageServer/Protocol/Features/Options/InlineHintsOptionsStorage.cs

```diff
@@ -31,78 +31,79 @@ public static InlineParameterHintsOptions GetInlineParameterHintsOptions(this IG
             };
 
         public static InlineTypeHintsOptions GetInlineTypeHintsOptions(this IGlobalOptionService globalOptions, string language)
-          => new()
-          {
-              EnabledForTypes = globalOptions.GetOption(EnabledForTypes, language),
-              ForImplicitVariableTypes = globalOptions.GetOption(ForImplicitVariableTypes, language),
-              ForLambdaParameterTypes = globalOptions.GetOption(ForLambdaParameterTypes, language),
-              ForImplicitObjectCreation = globalOptions.GetOption(ForImplicitObjectCreation, language),
-          };
+            => new()
+            {
+                EnabledForTypes = globalOptions.GetOption(EnabledForTypes, language),
+                ForImplicitVariableTypes = globalOptions.GetOption(ForImplicitVariableTypes, language),
+                ForLambdaParameterTypes = globalOptions.GetOption(ForLambdaParameterTypes, language),
+                ForImplicitObjectCreation = globalOptions.GetOption(ForImplicitObjectCreation, language),
+            };
 
-        private static readonly OptionGroup s_inlineHintOptionGroup = new(name: "inline_hints", description: "");
+        // Note: inlay hints is the term used in LSP, inline hints is the term used in VS. Want to use the LSP name when communicate with the LSP client.
```

Comment:
>inlay hints is the term used in LSP, inline hints is the term used in VS. Want to use the LSP name when communicate with the LSP client. [](http://example.com/codeflow?start=17&length=136)

I'd just say use "inlay" instead of "inline" to match LSP spec.

Reply:
Agreed.  I'd have us just follow LSP fully.